API/Controllers/HealthController.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[AllowAnonymous]
+public class HealthController : BaseApiController
+{
+
+    [HttpGet("")]
```

Comment:
Oh and remove the (""), as it's implied

Reply:
Fixed in [e22e75f](https://github.com/Kareadita/Kavita/pull/1480/commits/e22e75ff9d7e5ea4f6539018fa53f45ae341ce1e)